<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Administrator</class>
 <widget class="QWidget" name="Administrator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Administrator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Movies</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listMovies"/>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="titleLabel">
       <property name="text">
        <string>Title</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="titleLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="genreLabel">
       <property name="text">
        <string>Genre</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="genreLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="yearLabel">
       <property name="text">
        <string>Year</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="yearLineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="likesLabel">
       <property name="text">
        <string>Likes</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="likesLineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="trailerLabel">
       <property name="text">
        <string>Trailer</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="trailerLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updateButton">
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>addButton</sender>
   <signal>clicked()</signal>
   <receiver>Administrator</receiver>
   <slot>addMovie()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>680</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>699</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteButton</sender>
   <signal>clicked()</signal>
   <receiver>Administrator</receiver>
   <slot>deleteMovie()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>672</y>
    </hint>
    <hint type="destinationlabel">
     <x>391</x>
     <y>700</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>updateButton</sender>
   <signal>clicked()</signal>
   <receiver>Administrator</receiver>
   <slot>updateMovie()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>599</x>
     <y>681</y>
    </hint>
    <hint type="destinationlabel">
     <x>626</x>
     <y>697</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addMovie()</slot>
  <slot>deleteMovie()</slot>
  <slot>updateMovie()</slot>
 </slots>
</ui>
